src/widgets.cpp

```diff
@@ -2310,8 +2312,9 @@ class AFDataLifetimeArrayDialog : public wxDialog
 		{
 			wxFileDialog dlg(this, "Select data file to import");
 			if (dlg.ShowModal() != wxID_OK) return;
-			FILE *fp = fopen(dlg.GetPath().c_str(), "r");
-			if (!fp)
+			std::ifstream fp;
```

Comment:
Tested by @dguittet - should be fine

Reply:
If this PR can be approved and merged, I can do back and fix all remaining instances of `FILE *` uses.